run_md.py

```diff
@@ -0,0 +1,879 @@
+        try:
+            var_md_dirs = []
+            deffnm = f'{deffnm_prev}_{mdtime_ns}'
+            # os.path.dirname(var_lig)
+            for res in calc_dask(continue_md_from_dir, wdir_to_continue_list, dask_client,
+                                 tpr=tpr_prev, cpt=cpt_prev, xtc=xtc_prev,
+                                 deffnm_prev=deffnm_prev, deffnm_next=deffnm, mdtime_ns=mdtime_ns, project_dir=project_dir):
+                if res:
+                    var_md_dirs.append(res)
+
+        finally:
+            dask_client.shutdown()
+        logging.info(f'Continue of simulation of {var_md_dirs} were successfully finished\n')
+
+    # Part 3. MD Analysis. Run on each cpu
+    dask_client = init_dask_cluster(hostfile=hostfile, n_tasks_per_node=ncpu, ncpu=ncpu)
+    try:
+        var_md_analysis_dirs = []
+        # os.path.dirname(var_lig)
+        for res in calc_dask(run_md_analysis, var_md_dirs,
+                             dask_client, deffnm=deffnm, mdtime_ns=mdtime_ns, project_dir=project_dir):
+            if res:
+                var_md_analysis_dirs.append(res)
+    finally:
+        dask_client.shutdown()
+
+    logging.info(f'\nAnalysis of md simulation of {var_md_analysis_dirs} were successfully finished\n')
+
+    if not not_clean_log_files:
+        if wdir_to_continue_list is None:
+            for f in glob(os.path.join(wdir_md, '*', '#*#')):
+                os.remove(f)
+        else:
+            for wdir_md in wdir_to_continue_list:
+                for f in glob(os.path.join(wdir_md, '#*#')):
+                    os.remove(f)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='''Run or continue MD simulation.\n
+    Allowed systems: Protein in water, Protein-Ligand-multiple Coenzymes, Protein-Ligand, Protein-multiple Coenzymes''')
+    parser.add_argument('-p', '--protein', metavar='FILENAME', required=False, type=filepath_type,
+                        help='input file of protein. Supported formats: *.pdb or gro')
+    parser.add_argument('-d', '--wdir', metavar='WDIR', default=None, type=filepath_type,
+                        help='Working directory. If not set the current directory will be used.')
+    parser.add_argument('-l', '--ligand', metavar='FILENAME', required=False, type=filepath_type,
+                        help='input file with compound. Supported formats: *.mol or sdf or gro')
+    parser.add_argument('--cofactor', metavar='FILENAME', default=None, type=filepath_type,
+                        help='input file with compound. Supported formats: *.mol or sdf or gro')
+    parser.add_argument('--gromacs_version', metavar='STRING', default='GROMACS/2021.4-foss-2020b-PLUMED-2.7.3',
+                        help='gromacs_version')
+    parser.add_argument('--not_add_H', action='store_true', default=False,
+                        help='disable to add hydrogens to molecules before simulation.')
```

Comment:
It is unclear whether it is related to all supplied molecules or only to ligands/cofactors?
Do we need this argument at all? If ligands were supplied with hydrogens, we will not be able to add new hydrogens, because valence of atoms is full.